controllers/apps/cluster_controller.go

```diff
@@ -172,6 +172,8 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			&FillClassTransformer{},
```

Comment:
check `SetupWithManager` for Owns objects

Reply:
If the user does not grant RBAC access to KubeBlocks, and the manager owns the RBAC kinds, it can potentially lead to issues in this scenario.

Reply:
it's needed, check backup_controller as example on how to dynamically setup